src/main/java/org/folio/inventory/support/http/client/OkapiHttpClient.java

```diff
@@ -150,6 +154,10 @@ private void okapiHeaders(HttpClientRequest request) {
       request.headers().add(TOKEN_HEADER, this.token);
     }
 
+    if (this.requestId != null) {
```

Comment:
This is inconsistent, it should use the same logic as the token above:

    if (StringUtils.isNotBlank(this.requestId)) {

Reply:
I don't agree. This code is faster and uses a very common idiom.. Null value for absent header.